domains/Accounts/Controllers/AccountsProfileController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Resources\UserResource;
+use Illuminate\Http\Response;
+use Illuminate\Contracts\Auth\Factory as Auth;
+
+class AccountsProfileController extends Controller
+{
+    protected $auth;
+
+    public function __construct(Auth $auth)
+    {
+        $this->auth = $auth;
+    }
```

Comment:
No need for this. We can use the authenticated user instance that's present in the request. We just need to inject that instance of `Request` in the `__invoke()` method, below, and call `$request->user()`.

Reply:
Yes, is true!